internal/commands/app/create.go

```diff
@@ -316,7 +316,7 @@ func (cmd CommandCreate) handleCreateApp(
 		return err
 	}
 
-	if err := appLocal.Load(); err != nil {
+	if err := appLocal.LoadData(rootDir); err != nil {
```

Comment:
[nit] I think it's the same result, but could we use `appLocal.LoadData(appLocal.RootDir)` here?